maven/hw05-Annotations/src/main/java/ru/otus/Annotations/ContainerTest.java

```diff
@@ -0,0 +1,98 @@
+package ru.otus.Annotations;
+
+public class ContainerTest {
+    private long ErrorCounter = 0;
```

Comment:
```suggestion
    private long errorCounter = 0;
```

В java поля, переменные, методы принято называть с маленькой буквы.
Классы, интерфейсы с большой.

https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#367

В других местах об этом уже не писал, сами посмотрите.